coding-challenges.js

```diff
@@ -2,27 +2,40 @@
 
 // --------------------1) Create a function that takes in an array. Each time the function is run, remove the first item from the array and shuffle the remaining content. If the array is empty, return "The array is empty."
 
-
-
-
-
 var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example1 (can be a different order): ["yellow", "blue", "pink", "green"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
+const something = (array) => {
+	let newArray = []
+	array.slice(0,1);
+	return newArray
+	
+}
```

Comment:
hmmm not sure what this function is doing. right now it just returns an empty array `newArray`. let me know if you want to walk through this one.